shell/platform/common/cpp/text_input_model.cc

```diff
@@ -30,30 +30,30 @@ bool IsTrailingSurrogate(char32_t code_point) {
 
 }  // namespace
 
-TextInputModel::TextInputModel()
-    : selection_base_(text_.begin()), selection_extent_(text_.begin()) {}
+TextInputModel::TextInputModel() : selection_base_(0), selection_extent_(0) {}
```

Comment:
Nit: you can just initialize these at the declaration site now.

Reply:
Done.